packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/hooks/useFormatTimeStamp.js

```diff
@@ -0,0 +1,15 @@
+import { useIntl } from 'react-intl';
+
+const useFormatTimeStamp = (value) => {
```

Comment:
Would be nice to write a small unit test for this hook

Reply:
this file is already covered in parent test cases, here you can see the coverage.
<img width="872" alt="Screenshot 2022-11-25 at 18 38 13" src="https://user-images.githubusercontent.com/7886244/204034562-b6e6f5e8-2aac-4315-acd5-35b248f8789b.png">